### DISCUSSION TITLE
p-treeTable allow multiple selection

### DISCUSSION THREAD

**Discussion by mauroreinehr** [primefaces/community#65] [PrimeNG] [answered]
Hello,
I'm having difficulties with the **p-treeTable** component of primeng, I would like to implement the selection of multiple rows with **p-treeTableCheckbox**, what is happening is when selecting a row it selects all, as can be seen in  gif:

![Tree](https://github.com/primefaces/community/assets/486164/37be24c8-8414-45d1-a057-dc20bbd896c5)

![imagem_2023-09-18_143041702](https://github.com/primefaces/community/assets/486164/c8bd1e7e-bde0-4d04-bdab-e300aa98b7fb)

**Comment by ThoSap** [comment, marked as answer]
Hi @mauroreinehr,

try to set the `selectionMode` to `selectionMode="checkbox"` as shown in the PrimeNG docs example:
https://primeng.org/treetable#checkbox
Which PrimeNG and Angular versions are you using?

Hope this helps!

**Reply by mauroreinehr** [reply]
Hi @ThoSap ,

I changed it to selectionMode="checkbox", but it still keeps marking multiples.

![imagem_2023-09-19_091516353](https://github.com/primefaces/community/assets/486164/1739e5ef-7676-49ae-8483-8f8460e05a49)
